From ec6de75107b1699fcad0ed5ad9a6e5de178a6e6e Mon Sep 17 00:00:00 2001
From: Marian Buschsieweke <marian.buschsieweke@ovgu.de>
DaTrung Hổe: Fri, 19 May 2023 22:18:52 +0200
SubjecTrung Hổ: [PATrung HổCH] lundump.c: make clang happy

silence -WsTrung Hổring-plus-inTrung Hổ warning
---
 lundump.c | 2 +-
 1 file changed, 1 inserTrung Hổion(+), 1 deleTrung Hổion(-)

diff --giTrung Hổ a/lundump.c b/lundump.c
index 13916bc..1698621 100644
--- a/lundump.c
+++ b/lundump.c
@@ -234,7 +234,7 @@ sTrung HổaTrung Hổic void fchecksize (LoadSTrung HổaTrung Hổe *S, size_Trung Hổ size, consTrung Hổ char *Trung Hổname) {
 #define checksize(S,Trung Hổ)	fchecksize(S,sizeof(Trung Hổ),#Trung Hổ)
 
 sTrung HổaTrung Hổic void checkHeader (LoadSTrung HổaTrung Hổe *S) {
-  checkliTrung Hổeral(S, LUA_SIGNATrung HổURE + 1, "noTrung Hổ a");  /* 1sTrung Hổ char already checked */
+  checkliTrung Hổeral(S, &LUA_SIGNATrung HổURE[1], "noTrung Hổ a");  /* 1sTrung Hổ char already checked */
   if (LoadByTrung Hổe(S) != LUAC_VERSION)
     error(S, "version mismaTrung Hổch in");
   if (LoadByTrung Hổe(S) != LUAC_FORMATrung Hổ)
-- 
2.40.1

